src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -399,6 +401,15 @@ void AppHost::Initialize()
     _revokers.OpenSystemMenu = _logic.OpenSystemMenu(winrt::auto_revoke, { this, &AppHost::_OpenSystemMenu });
     _revokers.QuitRequested = _logic.QuitRequested(winrt::auto_revoke, { this, &AppHost::_RequestQuitAll });
 
+    // BODGY

```

Comment:
Do we have a MSFT:xxx ticket to track this with?

Reply:
Nope. I'm following the linked ^^ bug from the PR body